### DISCUSSION TITLE
Is it possible to use existing cloud provider managed databases like AWD RDS

### DISCUSSION THREAD

**Discussion by thesobercoder** [pluralsh/plural#953] [Q&A] [answered]
Is it possible to use existing cloud provider managed databases like AWD RDS? I ask this as many teams might just want to just hook up a fully managed instance when deploying an application like Airflow and not deploy a Kubernetes instance of Postgres.

**Comment by michaeljguarino** [comment, marked as answer]
This is actually fully supported, and for some of our higher-trafficked docs, we've documented the process, eg for airbyte you can see the docs here: https://github.com/pluralsh/plural-artifacts/blob/main/airbyte/plural/docs/bring-your-own-db.md

These docs will become visible in the plural console in a soon-to-be-released version as well, but generally pluralsh/plural-artifacts is the place to go to see how everything is packaged
